topological_navigation/scripts/navigation.py

```diff
@@ -150,6 +151,8 @@ def executeCallback(self, goal):
         if self.monit_nav_cli :
             self.cancelled = False
             self.preempted = False
+            self.no_orientation = goal.no_orientation
+            print "NO ORIENTATION (%s)" %self.no_orientation
```

Comment:
although, @Jailander, I really want to urge you to *not* use `print` but `rospy.loginfo` or similar (for reasons that should be obvious)